src/lib/import-export/export/export-database.ts

```diff
@@ -32,7 +33,11 @@ export async function exportDatabaseToFile(
 	}
 
 	// Move the file to its final destination
-	await rename( tempFilePath, finalDestination );
+	if ( isLinux() ) {
+		await move( tempFilePath, finalDestination );
```

Comment:
Would it work if we used` move` for all systems?

Reply:
Probably, but I'm not sure if there is any specific reason that rename was used.

Reply:
@jeroenpf , it seems that you are responsible of choosing the `rename` command. Was there any specific reason for this decision?

Reply:
I talked with Jeroen and it seems that there was not a particular reason. I will default that to all OSes.
I'll keep the `isLinux` though, for consistency.

Reply:
@wojtekn , we now use `move` everywhere instead of `rename`.